includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -467,6 +467,7 @@ public function get_settings(): WP_REST_Response {
 		return new WP_REST_Response(
 			[
 				'enabled_payment_method_ids'          => $enabled_payment_methods,
+				'available_payment_gateways'          => WC()->payment_gateways->get_available_payment_gateways(),
```

Comment:
Not sure if this is going to work, but can we consider something like `'is_card_present_eligible'            => $this->wcpay_gateway->is_card_present_eligible() && isset( WC()->payment_gateways->get_available_payment_gateways()['cod'] ),`?

Reply:
Good point @kalessil . I checked the usages of the `is_card_present_eligible` flag. Apart from `ManualCaptureControl` component, it is used to conditionally show the `Card Readers` entry in the Payments menu: https://github.com/Automattic/woocommerce-payments/blob/92525c2a637bf592ec412bb0a979ab91862575d1/includes/admin/class-wc-payments-admin.php#L399

So I removed the 'available_payment_gateways' in favor of your suggestion in this commit: https://github.com/Automattic/woocommerce-payments/pull/7638/commits/e30e49732d875f7a461400fc09abf7f3e0283401. For more context, I was wrongly assuming that the `is_card_present_eligible` was used as a feature flag, but now I see it is not the case. Thanks!